### DISCUSSION TITLE
Usage with Apple M1 possible?

### DISCUSSION THREAD

**Discussion by bmaciag** [fastlane/fastlane#17629] [Q&A] [answered]
Hi, I use an Apple M1 and when I run Fastlane I'm getting the following error:
```
CompileSwift normal x86_64 /Users/.../Category.swift (in target 'iOS' from project 'sPro')
[...]
Command CompileSwift failed with a nonzero exit code
```

What I also found in the log:
```
[...]
Testing failed:
	cdtool cannot compile [0]
	Command CompileSwift failed with a nonzero exit code
	Testing cancelled because the build failed.

** TEST FAILED **
```

How to check what's wrong here?

**Comment by max-ott** [comment, marked as answer]
As far as I know there are a lot of dependancies which are not working / have known issues with Apple Silicon. Would also be helpful to post this as an issue with full stack trace, context, commands you are running.

**Reply by bmaciag** [reply]
Will do so

**Comment by tristangrichard** [comment]
@bmaciag Did you manage to fix this?
